### PR TITLE
Adding photons to L1Ntuples

### PR DESCRIPTION
**Pull request by pallabidas**
#### PR description:

Adding reco::photon collection with MVA ID in L1Ntuples.

#### PR validation:

Tested locally with cmsDriver command.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport.

@bundocka @aloeliger

### CODE REVIEW

File: L1Trigger/L1TNtuples/plugins/L1PhotonRecoTreeProducer.cc
Comment (aloeliger) [inline L1Trigger/L1TNtuples/plugins/L1PhotonRecoTreeProducer.cc:1]: You may want to consider a `fillDescriptions` for any configuration defaults.

File: L1Trigger/L1TNtuples/python/L1NtupleAOD_cff.py [hunk 1]
```diff
@@ -17,7 +18,8 @@
   +l1JetRecoTree
   +l1MetFilterRecoTree
   +l1ElectronRecoTree
-  +l1TauRecoTree
+  +l1PhotonRecoTree
+#  +l1TauRecoTree
```
[the remark this answers is not in the document]
Comment (pallabidas) [reply L1Trigger/L1TNtuples/python/L1NtupleAOD_cff.py]: Added back.

File: L1Trigger/L1TNtuples/plugins/L1PhotonRecoTreeProducer.cc
Comment (pallabidas) [reply L1Trigger/L1TNtuples/plugins/L1PhotonRecoTreeProducer.cc:1]: Added.

**Comment by aloeliger** [comment]
please test

**Comment by aloeliger** [comment]
+l1

**Pull request merged**